src/server/game/DataStores/DBCStructure.h

```diff
@@ -926,6 +926,15 @@ struct EmotesTextEntry
     uint32  textid;
 };
 
+struct EmotesTextSoundEntry
+{
+    uint32 Id;                                              // 0
+    uint32 emotesTextId;                                    // 1
```

Comment:
Because that we actually have (some) coding standards: https://trinitycore.atlassian.net/wiki/pages/viewpage.action?pageId=2130103#C++DevelopmentStandards-Namingofvariablesandfunctions

The keyword there is "mostly". The other ones will need to be renamed, eventually.